Documentation/metadata-service.md

```diff
@@ -14,10 +14,11 @@ For systemd based distributions, it also supports the [systemd socket activation
 If using socket activation, keep the socket named /run/rkt/metadata-svc.sock as `rkt run` uses this name during registration.
 Please note that when started under socket activation, the metadata service will not remove the socket on exit.
 Use `RemoveOnStop` directive in .socket file to cleanup.
+`rkt-metadata.service` and `rkt-metadata.socket` are available in the `dist/init/systemd` directory of rkt project.
```

Comment:
Example systemd unit files for running the metadata service are available in [dist](dist/init/systemd)